From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@msolhab__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mohabib). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
u[pdates to ssl docs

Commit:
updates to ssl and waf rules

Commit:
updates to article , corrected links

Commit:
updates to the ssl docs

Commit:
attempt to correct missing bookmark issue

Commit:
fixed merge conflict

Commit:
updates

Commit:
updates to formatting

Commit:
url route update

Commit:
updates to articles

andretms committed:
AppSource Page Update

andretms committed:
Updates to manage style/ conflicts

andretms committed:
Trying to fix conflicts

andretms committed:
Trying to manage conflicts

andretms committed:
Attempt 3 to try to manage conficts

andretms committed:
Returning trial experience managing conflicts

andretms committed:
Removing links from previous version

andretms committed:
Update datetime

andretms committed:
Image scale in appsource page

andretms committed:
Managing conflicts

andretms committed:
Dealing with conflicts

andretms committed:
Dealing with conflicts

andretms committed:
Other conflicts fighting

andretms committed:
More attempts to deal with conflicts

andretms committed:
Other conflict fighting

andretms committed:
More conflict handling

andretms committed:
Content update

andretms committed:
Fixing warnings for hyperlinks

andretms committed:
Updating hyperlink

andretms committed:
Updating images

andretms committed:
Image scale resize

andretms committed:
Resizing images, table

andretms committed:
Update to screen shot

andretms committed:
Testing small changes in layout

andretms committed:
Small content fixes

andretms committed:
v2 overview - current version

andretms committed:
Revert "v2 overview - current version"

This reverts commit 62c7c1518488c6fae68142d7e593c09ab2aff358.

andretms committed:
Updating the header to have updated date

andretms committed:
Updates to appSource page based on feedback

andretms committed:
Resolving conflicts

andretms committed:
Minor changes

andretms committed:
Minor changes based on Acrolinx review

andretms committed:
Small changes in page

MarkMcGeeAtAquent committed:
Merge pull request #3 from MicrosoftDocs/master

update

MarkMcGeeAtAquent committed:
Clarify output type #11201

MarkMcGeeAtAquent committed:
Added punctuation

MarkMcGeeAtAquent committed:
Remove en-us

MarkMcGeeAtAquent committed:
branded CLI

WilliamAntonRohm committed:
Merge pull request #19 from MicrosoftDocs/master

bring up to date

Commit:
updates to articles

Commit:
merge conflict

Commit:
updates to cli scripts

andretms committed:
Fixing Typo

WilliamAntonRohm committed:
Merge pull request #20 from MicrosoftDocs/master

bringing in the newest updates

Commit:
app gateway : tabbed content

Commit:
updates

Commit:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into appgatewaycli20

Commit:
updates to fix merge conflict

Commit:
updates to cli articles

MarkMcGeeAtAquent committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into July24

Commit:
acrolinx updates

Commit:
removing inaccurate cli in powershell article

Commit:
corrected a mispelling

WilliamAntonRohm committed:
Merge pull request #21 from MicrosoftDocs/master

bringing in the newbies

andretms committed:
Update based on feedback

andretms committed:
Some updates in wording

WilliamAntonRohm committed:
Merge pull request #22 from MicrosoftDocs/master

updating my fork

johnpapa committed:
[aci] - deploy container - add jmespath queries

these queries help show the state of the deployment and the ip address without all of the dozens of other json lines. This is much easier to follow for the customer.

johnpapa committed:
Update container-instances-tutorial-deploy-app.md

cephalin committed:
AI update

cephalin committed:
acrolinx

cephalin committed:
acrolinx

cephalin committed:
buydomain feedback

cephalin committed:
update dates

Commit:
updates

anandyadavMSFT committed:
adding note for multi-domain federation sync warning

anandyadavMSFT committed:
spn note

Commit:
corrected merge conflicts

Commit:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into appgatewaycli20

anandyadavMSFT committed:
updating SPN text

anandyadavMSFT committed:
updating note text for multiple federation

Commit:
cli docs

ggailey777 committed:
Collapsed two tasks

ggailey777 committed:
Remove extra space

Commit:
rewriting slightly

WilliamAntonRohm committed:
Merge pull request #23 from MicrosoftDocs/master

bringing the changes home

WilliamAntonRohm committed:
minor code fixes

Commit:
updates

ktoliver committed:
Merge pull request #18632 from Blackmist/fresh2

updates

andretms committed:
Text updates

ggailey777 committed:
Replaced X with "supported"

Commit:
Removing EULAs

andretms committed:
Additional fixes, typos

Commit:
Merge pull request #18644 from WilliamAntonRohm/W-a-dotnet

minor updates

markgalioto committed:
couple of updates to correct wording of around billing details

Commit:
Merge pull request #18635 from anandyadavMSFT/updatemultifed

adding note for multi-domain federation sync warning

Commit:
Merge pull request #18645 from ggailey777/patch-2

Replaced X with "supported"

Commit:
Merge pull request #18643 from ggailey777/patch-1

Collapse two steps into a single task

ktoliver committed:
Merge pull request #15130 from georgewallace/appgatewaycli20

application gateway - cli 2.0 updates

Commit:
Merge pull request #18039 from MarkMcGeeAtAquent/July24

Added note to clarify output type

Commit:
Merge pull request #18647 from markgalioto/Az8-3

couple of updates to correct wording of around billing details

kgremban committed:
Cancelled carrot characters

douglaslMS committed:
Updated 2 PS Data Sync scripts with PM updates.

Commit:
Merge pull request #18650 from kgremban/patch-1

Cancelled carrot characters

Commit:
Merge pull request #18651 from douglaslMS/datasync-update-scripts

Updated 2 PS Data Sync scripts with PM updates.

cephalin committed:
reorder

andretms committed:
Merge from main

andretms committed:
Fixing document date

andretms committed:
Addressing acrolinks issues

twooley committed:
added info about portal link to forums

Ja-Dunn committed:
Merge pull request #18628 from cephalin/troubleshoot

AI update

Commit:
Merge pull request #18652 from twooley/twsupport

added info about portal link to forums

johnpapa committed:
Update container-instances-tutorial-deploy-app.md

refiewer comments addressed

ShilpiGautam committed:
Update swagger endpoint

MGoedtel committed:
removed statement regarding only available from classic portal

Ja-Dunn committed:
Merge pull request #17480 from andretms/AppSource

App source Content Update

Commit:
Merge pull request #18658 from MGoedtel/UpdateLAUpgrade832017

removed statement regarding only available from classic portal

Ja-Dunn committed:
Merge pull request #18627 from johnpapa/patch-2

[aci] - deploy container - add jmespath queries

Commit:
Merge pull request #18654 from ShilpiGautam/patch-1

Update swagger endpoint

Lisaco88 committed:
Merge pull request #18646 from Minewiskan/remeulas

Removing EULAs

neilpeterson committed:
updated command

Commit:
Merge pull request #18663 from neilpeterson/acs-tutorial-patch

updated command

cherylmc committed:
update client configuration package

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

MicrosoftGuyJFlo committed:
Take Ownership to Delete

Commit:
Merge pull request #18666 from cherylmc/updatedns

update client configuration package

v-alje committed:
Merge pull request #18667 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #18668 from MicrosoftGuyJFlo/SaaSAppTutorialDuplicatePublish

Take Ownership to Delete
